plugin/src/test/groovy/com/github/triplet/gradle/play/GenerateResourcesTest.groovy

```diff
@@ -367,4 +367,46 @@ class GenerateResourcesTest {
         assertEquals(originalFullDescription, processedFullDescription)
         assertEquals(originalTitle, processedTitle)
     }
+
+    @Test
+    void languageMerge() {
+        def project = TestHelper.evaluatableProject()
+        def originalTitle = new File(TestHelper.FIXTURE_WORKING_DIR,
+                'src/main/play/listings/en-US/title').text
+        def originalFullDescription = new File(TestHelper.FIXTURE_WORKING_DIR,
+                'src/main/play/listings/de-DE/fulldescription').text
+        def originalShortDescription = new File(TestHelper.FIXTURE_WORKING_DIR,
+                'src/staging/play/listings/en-US/shortdescription').text
+
+        project.android {
+            flavorDimensions 'pricing', 'server'
+
+            productFlavors {
+                free { dimension 'server' }
+                paid { dimension 'pricing' }
+                staging { dimension 'server' }
+                prod { dimension 'pricing' }
```

Comment:
adding prod flavor to pricing dimension. Is that on purpose to be able to run the desired check?

Reply:
Yeah, I had to move the dimensions and flavors around to get the overrides (or lack thereof) I wanted.